.github/workflows/test-coverage.yaml

```diff
@@ -27,5 +27,5 @@ jobs:
           needs: coverage
 
       - name: Test coverage
-        run: covr::codecov(quiet = FALSE)
+        run: covr::codecov(function_exclusions = "^make", quiet = FALSE)
```

Comment:
As both `make_json_*()` functions write to file I didn't want them running everytime the tests run and the code coverage was calculated. I could write tests and skip them if you'd prefer?

Reply:
One solution could be to create a temporary path from the test file, and read/write to/from that. See e.g. https://github.com/epiverse-trace/scenarios/blob/main/tests/testthat/test-import_export.R. That would allow you test all functions.

Reply:
Thanks, I'll add some tests using the temporary path.

Reply:
Resolved in ffff318